From bef89d989994955e5403c7fd090c5dc786a6fd3c Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Sun, 20 Feb 2022 13:59:00 +0100
Subject: [PATCH] treewide: rename Turris 1.x target to more accurate name

This is required becase of changes done in Turris Build repository.
The previous target was not good to upstream it to OpenWrt. This helps a
little bit to be in sync how the Turris Omnia and Turris MOX are done
---
 hardware/atsha204/getmac/Makefile            | 2 +-
 hardware/atsha204/libatsha204/Makefile       | 4 ++--
 hardware/cert-backup/Makefile                | 2 +-
 hardware/turris1x/turris1x-btrfs/Makefile    | 2 +-
 hardware/turris1x/turris1x-firmware/Makefile | 2 +-
 hardware/turris1x/turris1x-rainbow/Makefile  | 2 +-
 hardware/turris1x/turris1x-support/Makefile  | 2 +-
 hardware/turris1x/turris1x-uboot/Makefile    | 2 +-
 8 files changed, 9 insertions(+), 9 deletions(-)

diff --git a/hardware/atsha204/getmac/Makefile b/hardware/atsha204/getmac/Makefile
index de1029e7c..24993e87f 100644
--- a/hardware/atsha204/getmac/Makefile
+++ b/hardware/atsha204/getmac/Makefile
@@ -22,7 +22,7 @@ include $(INCLUDE_DIR)/package.mk
 
 define Package/getmac
   TITLE:=Get MAC
-  DEPENDS:=@TARGET_mpc85xx_p2020_DEVICE_turris1x
+  DEPENDS:=@TARGET_mpc85xx_p2020_DEVICE_cznic_turris1x
 endef
 
 define Build/Compile
diff --git a/hardware/atsha204/libatsha204/Makefile b/hardware/atsha204/libatsha204/Makefile
index 4ebcc94c6..2ff8ca5ac 100644
--- a/hardware/atsha204/libatsha204/Makefile
+++ b/hardware/atsha204/libatsha204/Makefile
@@ -27,7 +27,7 @@ define Package/libatsha204
   URL:=https://gitlab.nic.cz/turris/libatsha204
   PROVIDES:=turris-otp
   DEPENDS:=+libopenssl +libunbound
-  DEPENDS+=@(TARGET_mvebu_cortexa9_DEVICE_cznic_turris-omnia||TARGET_mpc85xx_p2020_DEVICE_turris1x)
+  DEPENDS+=@(TARGET_mvebu_cortexa9_DEVICE_cznic_turris-omnia||TARGET_mpc85xx_p2020_DEVICE_cznic_turris1x)
   VARIANT:=real
 endef
 
@@ -45,7 +45,7 @@ ifeq ($(BUILD_VARIANT),real)
   MAKE_FLAGS += USE_LAYER=USE_LAYER_NI2C
   ifeq ($(CONFIG_TARGET_mvebu_cortexa9_DEVICE_cznic_turris-omnia),y)
     MAKE_FLAGS += DEFAULT_NI2C_DEV_PATH=NI2C_DEV_PATH_OMNIA
-  else ifeq ($(CONFIG_TARGET_mpc85xx_p2020_DEVICE_turris1x),y)
+  else ifeq ($(CONFIG_TARGET_mpc85xx_p2020_DEVICE_cznic_turris1x),y)
     MAKE_FLAGS += DEFAULT_NI2C_DEV_PATH=NI2C_DEV_PATH_TURRIS
   endif
 endif
diff --git a/hardware/cert-backup/Makefile b/hardware/cert-backup/Makefile
index 3e19534ed..f3ec17af9 100644
--- a/hardware/cert-backup/Makefile
+++ b/hardware/cert-backup/Makefile
@@ -42,7 +42,7 @@ endef
 define Package/cert-backup-turris1x
   $(call Package/CommonBackend)
   TITLE+= - Turris 1.x backend
-  DEPENDS:=@TARGET_mpc85xx_p2020_DEVICE_turris1x
+  DEPENDS:=@TARGET_mpc85xx_p2020_DEVICE_cznic_turris1x
   CONFLICTS:=cert-backup-btrfs
 endef
 
diff --git a/hardware/turris1x/turris1x-btrfs/Makefile b/hardware/turris1x/turris1x-btrfs/Makefile
index 9681bbde6..4e9851f3c 100644
--- a/hardware/turris1x/turris1x-btrfs/Makefile
+++ b/hardware/turris1x/turris1x-btrfs/Makefile
@@ -20,7 +20,7 @@ define Package/Common
   SECTION:=utils
   CATEGORY:=Utilities
   SUBMENU:=Filesystem
-  DEPENDS:=@TARGET_mpc85xx_p2020_DEVICE_turris1x
+  DEPENDS:=@TARGET_mpc85xx_p2020_DEVICE_cznic_turris1x
 endef
 
 define Package/turris1x-btrfs
diff --git a/hardware/turris1x/turris1x-firmware/Makefile b/hardware/turris1x/turris1x-firmware/Makefile
index 09cb73b7c..b4212e784 100644
--- a/hardware/turris1x/turris1x-firmware/Makefile
+++ b/hardware/turris1x/turris1x-firmware/Makefile
@@ -17,7 +17,7 @@ include $(INCLUDE_DIR)/package.mk
 
 define Package/turris-1x-firmware
   TITLE:=Firmware for Turris 1.x
-  DEPENDS:=@TARGET_mpc85xx_p2020_DEVICE_turris1x
+  DEPENDS:=@TARGET_mpc85xx_p2020_DEVICE_cznic_turris1x
   PROVIDES:=turris-nor-update-firmware
 endef
 
diff --git a/hardware/turris1x/turris1x-rainbow/Makefile b/hardware/turris1x/turris1x-rainbow/Makefile
index aaac87bfc..028230b5b 100644
--- a/hardware/turris1x/turris1x-rainbow/Makefile
+++ b/hardware/turris1x/turris1x-rainbow/Makefile
@@ -26,7 +26,7 @@ define Package/turris1x-rainbow
   TITLE:=turris1x-rainbow
   URL:=https://gitlab.nic.cz/turris/rainbow
   PROVIDES:=turris-rainbow
-  DEPENDS:=@KERNEL_DEVMEM @TARGET_mpc85xx_p2020_DEVICE_turris1x
+  DEPENDS:=@KERNEL_DEVMEM @TARGET_mpc85xx_p2020_DEVICE_cznic_turris1x
 endef
 
 define Package/turris1x-rainbow/conffiles
diff --git a/hardware/turris1x/turris1x-support/Makefile b/hardware/turris1x/turris1x-support/Makefile
index 94286d8c2..7dcd9b412 100644
--- a/hardware/turris1x/turris1x-support/Makefile
+++ b/hardware/turris1x/turris1x-support/Makefile
@@ -15,7 +15,7 @@ include $(INCLUDE_DIR)/package.mk
 define Package/turris1x-support
   TITLE:=Turris 1.x hardware support files
   VERSION:=$(LINUX_VERSION)-$(LINUX_RELEASE)-$(LINUX_VERMAGIC)
-  DEPENDS:=@TARGET_mpc85xx_p2020_DEVICE_turris1x
+  DEPENDS:=@TARGET_mpc85xx_p2020_DEVICE_cznic_turris1x
 endef
 
 define Package/turris1x-support/description
diff --git a/hardware/turris1x/turris1x-uboot/Makefile b/hardware/turris1x/turris1x-uboot/Makefile
index 97dc24c6e..f41ec06af 100644
--- a/hardware/turris1x/turris1x-uboot/Makefile
+++ b/hardware/turris1x/turris1x-uboot/Makefile
@@ -17,7 +17,7 @@ include $(INCLUDE_DIR)/package.mk
 
 define Package/turris1x-uboot
   TITLE:=Turris 1.x U-boot
-  DEPENDS:=@TARGET_mpc85xx_p2020_DEVICE_turris1x
+  DEPENDS:=@TARGET_mpc85xx_p2020_DEVICE_cznic_turris1x
   PROVIDES:=u-boot-turris1x
 endef
 
-- 
2.34.1

